views/partials/_example.njk

```diff
@@ -4,7 +4,7 @@
 {% set exampleURL = "/" + params.group + "/" + params.item + "/" + params.example + "/index.html" %}
 {% set examplePrefix = params.item + "-" + params.example + "-" %}
 
-<div class="app-c-example app-c-example--tabs">
+<div class="app-c-example{% if (params.html and params.nunjucks) %} app-c-example--tabs{% endif %}">
```

Comment:
I made this change originally because the `app-c-example--tabs` sets `margin-bottom: 0` on the example, which meant _when the prose scope was applied_ and headings no longer have a top margin there was no spacing at all between the examples and the next heading.

However, if we're no longer applying the prose scope as part of this PR then it probably does makes sense to defer this change until then.

Reply:
Fair points. I've removed the commit from this PR.